classtest.js

```diff
@@ -1,87 +1,152 @@
 'use strict';
-
-//-------------------------VARIABLES-------------------------
 var timeSlot = {
   label: 'Times: ',
   times: ['6am|', '7am|', '8am|', '9am|', '10am|', '11am|', '12pm|', '1pm|', '2pm|', '3pm|', '4pm|', '5pm|', '6pm|', '7pm|'],
   totalLabel: 'Total|'
 };
-var pikePlace = new CookieStore ('First and Pike:', 23, 65, 6.3);
-// console.log(pikePlace.displayCookies());
-var seaTac = new CookieStore ('SeaTac:', 3, 24, 1.2);
-// console.log(seaTac.displayCookies());
-var seattleCenter = new CookieStore('Seattle Center:', 11, 38, 3.7);
-// console.log(seattleCenter.displayCookies());
-var capHill = new CookieStore('Capitol Hill:', 20, 38, 2.3);
-// console.log(capHill.displayCookies());
-var alki = new CookieStore('Alki:', 2, 16, 4.6);
-// console.log(alki.displayCookies());
-var tableEl = document.createElement('table');
+  //---New store constructors--//
+var pikePlace = new CookieStore ('First and Pike', 23, 65, 6.3);
+var seaTac = new CookieStore ('SeaTac', 3, 24, 1.2);
+var seattleCenter = new CookieStore('Seattle Center', 11, 38, 3.7);
+var capHill = new CookieStore('Capitol Hill', 20, 38, 2.3);
+var alki = new CookieStore('Alki', 2, 16, 4.6);
+  //-- array of the stores for easy access and pulling the info---//
 var stores = [pikePlace, seaTac, seattleCenter, capHill, alki];
+var colTotals = [];
+  //---starts the total at zero so that we can calculate totals within the for loops. resetting to zero allows us to differentiate different rows---//
 var total = 0;
-//----------------------For Loop to run through the times------------
-
-var rowOneEl = document.createElement('tr');
-var anything = document.createElement('th');
-anything.textContent = timeSlot.label;
-rowOneEl.appendChild(anything);
-
-for (var k = 0; k < timeSlot.times.length ; k++){
-  var timeLabelEl = document.createElement('th');
-  var currentTime = timeSlot.times;
-  timeLabelEl.textContent = currentTime[k];
-  rowOneEl.appendChild(timeLabelEl);
-};
-var totalHead = document.createElement('th');
-totalHead.textContent = timeSlot.totalLabel;
-rowOneEl.appendChild(totalHead);
-
-tableEl.appendChild(rowOneEl);
-
+  //---Creates the table element in the html that we insert into-----//
+var tableEl = document.createElement('table');
 document.body.appendChild(tableEl);
 
-for(var i = 0; i < stores.length; i++){
-  var currentStore = stores[i];
+//------------------------FUNCTIONS----------------------------//
```

Comment:
Labelling sections like this is great! That way you can find stuff in the minimap.